src/main/java/liquibase/ext/databricks/sqlgenerator/CreateTableGeneratorDatabricks.java

```diff
@@ -1,22 +1,36 @@
 package liquibase.ext.databricks.sqlgenerator;
 
 
+import liquibase.database.Database;
 import liquibase.exception.ValidationErrors;
 import liquibase.ext.databricks.change.createTable.CreateTableStatementDatabricks;
 import liquibase.ext.databricks.database.DatabricksDatabase;
-import liquibase.sqlgenerator.core.CreateTableGenerator;
-import liquibase.database.Database;
 import liquibase.sql.Sql;
 import liquibase.sql.UnparsedSql;
 import liquibase.sqlgenerator.SqlGeneratorChain;
+import liquibase.sqlgenerator.core.CreateTableGenerator;
 import liquibase.statement.core.CreateTableStatement;
 import liquibase.structure.DatabaseObject;
 import org.apache.commons.lang3.StringUtils;
 
-import java.util.List;
+import java.util.*;
+
+import static java.util.stream.Collectors.joining;
 
 public class CreateTableGeneratorDatabricks extends CreateTableGenerator {
 
+    private static final Set<String> ESSENTIAL_PROPERTIES = Set.of(
```

Comment:
Could you use the map keys (like in `DEFAULT_VALUES.keySet() `) instead of having a `ESSENTIAL_PROPERTIES` with the same values?